puppeteer-test/src/tests/workspace/workspace-permission.spec.ts

```diff
@@ -0,0 +1,39 @@
+import PuppeteerLaunch from '../../services/puppeteer-launch';
```

Comment:
What are workspace-permission tests intended to be? It may not be worth keeping this around, if it's easy enough for folks to create the new file as needed.

Reply:
Deleted